project/interpolation_internal_test.go

```diff
@@ -0,0 +1,161 @@
+package project
```

Comment:
From what I've seen there is a convention to name the test file `interpolation_test.go`. What's the reason for deviating from that convention ?